Example/Backpack/View Controllers/TextFieldsViewController.swift

```diff
@@ -0,0 +1,31 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+import Backpack.TextField
+import Backpack.Color
+
+class TextFieldsViewController: UIViewController {
+    @IBOutlet weak var textField: TextField!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        textField.textColor = Color.red800
```

Comment:
Could we remove this line? As a general rule we make the examples use the default styling

Reply:
Sure! I just wanted to test that the color remains when the user edits :)